the-content-guide/readme.md

```diff
@@ -111,6 +111,18 @@ Most importantly, images enrich an article in the sense that they support the wr
 
 A good mantra, with the addition of more multimedia in the back of our minds, is not "form follows content" but "content follows form".
 
+### The use of movie stills
+
+Copyright law is a hassle and can cause headaches 🤕. Movie stills for example, can provide an article with valuable context. The copyright law of these creative works is complicated. Yet, there are some arguments that can reduce your stress level while using them. The actual movies are definitely copyrighted, but some argue that movie stills (a very small part of the whole creative work) fall under [*fair use*](http://fairuse.stanford.edu/overview/fair-use/what-is-fair-use/) material.  
+
+In 1993 an Ad Hoc Committee discussed this issue and argued that: They not only believe that it is *fair use* to publish film stills, they do not believe that gaining permission should be necessary. Report of the Ad Hoc Committee of the [Society For Cinema Studies](http://www.cmstudies.org/?page=fair_use), [Fair Usage Publication of Film Stills](http://www.intellectbooks.co.uk/MediaManager/File/Film%20Stills.pdf) by Kristen Thompson.
+
```

Comment:
I think we can just make  Ad Hoc Committee a link to the pdf and scrap the last sentence since it's too detailed/ doesn't add much to general readers.